docs/source/intro-clx-loda-anomaly-detection.ipynb

```diff
@@ -0,0 +1,171 @@
+{
```

Comment:
<p>can you elaborate or give an example of what a 5-D dataset might be in real life?</p><p> </p><p>is this 100 measurements each with 5 features?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>yes, it is 100 measurements each with 5 features</p><p> </p>

Reply:
<p>I've added the In depth example notebook link which uses more realistic dataset.</p>